bugbot/rules/security_unhide_dups.py

```diff
@@ -0,0 +1,75 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.bugzilla import Bugzilla
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+
+
+class SecurityUnhideDups(BzCleaner):
+    """Security bugs that could be un-hidden"""
+
+    def description(self):
+        return "Security bugs that are marked as duplicates of already-public bugs"
+
+    def filter_no_nag_keyword(self):
+        return False
+
+    def get_summary(self, bug):
+        # This will prevent the default behavior of hiding the summary of
+        # security bugs.
+        return bug["summary"]
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        data[bugid] = bug
```

Comment:
Nit: 
```suggestion
        data[str(bug["id"])] = bug
```

Reply:
Done in bb20bb358122391a93461f39122d467b32b9ff47. I used `bugid` as a kind of consistency with other rules.